docs/wiki/research/roadmap.md

```diff
@@ -17,43 +17,44 @@ The discussion, resources and all research and development on the core protocol
-| Merge Execution and Consensus Layers |              Ethereum's execution layer merged with the Beacon chain (consensus layer)              |                                             Proof of work activities ceased and the network's consensus mechanism shifted to proof of stake                                             | shipped                    |
-| Enable Withdrawals                   | The last of the three-part process of Ethereum's transition to a proof of stake consensus mechanism |                                        Validators can push withdrawals from the Beacon chain to the EVM via a new "system-level" operation type                                         | shipped </br>EIP-4895[^2]  |
+| Upgrade                              | Track |    Topic    |                                             Description                                             |                                                                                     Expected effect                                                                                     | State of the art                                     |
+| :----------------------------------- | :---: | :---------: | :-------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------- |
+| Launch the Beacon Chain              |   -   |    Merge    |             A crucial step in Ethereum's shift to a proof of stake consensus mechanism              | Beacon Chain becomes the engine of block production, replacing mining. Validators have the role and responsibility for processing the validity of all transactions and proposing blocks | shipped </br> EIP-2982<span markdown='1'>[^1]</span> |
+| Merge Execution and Consensus Layers |   -   |    Merge    |              Ethereum's execution layer merged with the Beacon chain (consensus layer)              |                                             Proof of work activities ceased and the network's consensus mechanism shifted to proof of stake                                             | shipped                                              |
+| Enable Withdrawals                   |   -   | Withdrawals | The last of the three-part process of Ethereum's transition to a proof of stake consensus mechanism |                                        Validators can push withdrawals from the Beacon chain to the EVM via a new "system-level" operation type                                         | shipped </br>EIP-4895[^2]                            |
```

Comment:
Not sure if 'push withdrawals from BN to EVM' makes sense here. Validators can add withdrawal credentials and protocol automatically withdraws all inactive capital. 

```suggestion
| Enable Withdrawals                   |   -   | Withdrawals | The last of the three-part process of Ethereum's transition to a proof of stake consensus mechanism |                                        Validators can add their withdrawal credentials and protocol continuously sweeps inactive capital from the validator set                       | shipped </br>EIP-4895[^2]                            |
```